query-node/schema.graphql

```diff
@@ -239,6 +238,19 @@ type Channel @entity {
   happenedIn: Block!
 }
 
+type CuratorGroup @entity {
+  "Runtime entity identifier (EntityId)"
+  id: ID!
+
+  "Curators belonging to this group"
+  curatorIds: [BigInt!]
+
+  "Is group active or not"
+  isActive: Boolean!
+
+  videos: [Video!] @derivedFrom(field: "ownerCuratorGroup")
```

Comment:
Shouldn't this be `channels` as well? It seems to be referring to the same thing as the field in `Membership`

Reply:
Correct, curators do not directly operate channels.

Reply:
I think they _do_ operate channels, didn't you mean they don't operate on `videos` directly?

Reply:
Yes, that was a mistake. Fixed